iroh-net/src/discovery/pkarr.rs

```diff
@@ -21,27 +63,56 @@ use crate::{
     key::SecretKey,
     AddrInfo, Endpoint, NodeId,
 };
+
 #[cfg(feature = "discovery-pkarr-dht")]
 pub mod dht;
 
-/// The pkarr relay run by n0, for production.
+/// The production pkarr relay run by [number 0].
+///
+/// This server is both a pkarr relay server as well as a DNS resolver, see the [module
+/// documentation].  However it does not interact with the Mainline DHT, so is a more
+/// central service.  It is a reliable service to use for node discovery.
+///
+/// [number 0]: https://n0.computer
+/// [module documentation]: crate::discovery::pkarr
 pub const N0_DNS_PKARR_RELAY_PROD: &str = "https://dns.iroh.link/pkarr";
-/// The pkarr relay run by n0, for testing.
+/// The testing pkarr relay run by [number 0].
+///
+/// This server operates similarly to [`N0_DNS_PKARR_RELAY_PROD`] but is not as reliable.
+/// It is meant for more experimental use and testing purposes.
+///
+/// [number 0]: https://n0.computer
 pub const N0_DNS_PKARR_RELAY_STAGING: &str = "https://staging-dns.iroh.link/pkarr";
 
-/// Default TTL for the records in the pkarr signed packet. TTL tells DNS caches
-/// how long to store a record. It is ignored by the iroh-dns-server as the home
-/// server keeps the records for the domain. When using the pkarr relay no DNS is
-/// involved and the setting is ignored.
+/// Default TTL for the records in the pkarr signed packet.
+///
+/// The Time To Live (TTL) tells DNS caches how long to store a record. It is ignored by the
+/// `iroh-dns-server`, e.g. as running on [`N0_DNS_PKARR_RELAY_PROD`], as the home server
+/// keeps the records for the domain. When using the pkarr relay no DNS is involved and the
+/// setting is ignored.
+// TODO(flub): huh?
```

Comment:
I also cannot say what the last sentence is supposed to mean.

This paragraph should likely instead explain that this is the timeframe in which, after publishing new node addr info, other nodes might still receive the old outdated info when resolving via DNS.